### PR TITLE
Added New Sites, Fixed one

### PR DESCRIPTION
**Pull request by ef1500**
Added Marshmallow, Twitcasting, and Sploder

Fixed Anilist

### CODE REVIEW

File: web_accounts_list.json [hunk 1]
```diff
@@ -1597,6 +1597,17 @@
         "category" : "business",
         "valid" : true
       },
+      {
+        "name": "Marshmallow",
+        "check_uri": "https://marshmallow-qa.com/{acccount}",
```
Comment (WebBreacher) [inline web_accounts_list.json:1602]: there is a typo here: `acccount` with 3 c's. please fix.

File: web_accounts_list.json [hunk 2]
```diff
@@ -2161,6 +2172,17 @@
         "category" : "social",
         "valid" : true
       },
+      {
+        "name" : "Sploder",
+        "check_uri" : "https://www.sploder.com/games/members/{account}",
```
Comment (WebBreacher) [inline web_accounts_list.json:2177]: I'm getting errors  on this site for the detection strings. Can you check and try again?

` !  ERROR: {'name': 'Sploder', 'check_uri': 'https://www.sploder.com/games/members/{account}', 'account_existence_code': '200', 'account_existence_string': 'Awesomeness', 'account_missing_code': '404', 'account_missing_string': 'The game you were looking for by', 'known_accounts': ['vaxen2', 'geoff'], 'category': 'gaming', 'valid': True} BAD CODE AND STRING. Neither the HTTP response code or detection string worked.`

**Review by WebBreacher — CHANGES_REQUESTED** [review]

File: web_accounts_list.json (hunk 2 above)
Comment (ef1500) [reply web_accounts_list.json]: Sploder was acting rather strange, so I just removed it altogether.

**Comment by WebBreacher** [comment]
Thanks for making those changes.

**Pull request merged**